GOVERNANCE.md

```diff
@@ -0,0 +1,61 @@
+## K8sNetworkPlumbingWG Governance
```

Comment:
i assume this adds on to[1] ?
perhaps (until its merged into one) we should mention it here.


[1] https://docs.google.com/document/d/1lIWOK-W6fb1VZiSjO1BoFUs0TmoVwgd4XuVc6mtJk2c/edit#heading=h.5x0d5h95i329

Reply:
I didn't even realise that doc existed 😆. I think we should try to merge this PR as is - I'll mention the other doc in it. I'll do a follow-up round where I add the actual info from that doc and we can deprecate the Google Doc version eventually.

Thanks for pointing that out!

Reply:
I've updated this to make it an amendment to the existing doc - thanks for pointing to that.